perf/stability/alertmanager/alertmanagerconfig.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+data: {}
+kind: Secret
+metadata:
+  name: alertmanager-main
+  namespace: istio-prometheus
+stringData:
+  alertmanager.yaml: |-
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['alertname']
+      group_wait: 10s
+      group_interval: 10s
+      repeat_interval: 1h
+      receiver: 'test'
+    receivers:
+      - name: 'test'
+        webhook_configs:
+          - url: 'http://am-webhook.istio-prometheus:5001/webhook'
+        slack_configs:
+          - api_url: "https://hooks.slack.com/services/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"
+            channel: "#stability-test"
+            send_resolved: true
+            title: |-
+              [{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ .CommonLabels.alertname }} for {{ .CommonLabels.job }}
```

Comment:
Can you also add mesh id / cluster id?

Reply:
done, the alerts are grouped by cluster now